packages/fluentui/react-northstar/src/components/Carousel/CarouselNavigation.tsx

```diff
@@ -137,7 +144,7 @@ export const CarouselNavigation: ComponentWithAs<'ul', CarouselNavigationProps>
             vertical,
             thumbnails,
           }),
-        overrideProps: handleItemOverrides(variables),
+        overrideProps: handleItemOverrides(variables, disableClickableNav),
```

Comment:
I wouldn't pass the variable here... You can either prevent the item click in the `renderNavigation` method like this:

```jsx
overrideProps: (predefinedProps: CarouselNavigationItemProps) => ({
          onItemClick: (e: React.SyntheticEvent, itemProps: CarouselNavigationItemProps) => {
              if(disableClickableNav) return;
          
            const { index } = itemProps;

            setActiveIndex(e, index, true);

            _.invoke(predefinedProps, 'onClick', e, itemProps);
          },
        }),
```

Or pass it down to them props and prevent de click in the `CarouselNavavigationItem`